cookbooks/simple_ice_box.prm

```diff
@@ -0,0 +1,174 @@
+# Material model
+# Rheology: Non-linear viscous flow and Drucker Prager Plasticity
+# Values for most rheological parameters are specified for a background material and
+# each compositional field.  Values for viscous deformation are based on diffusion creep 
+# based on Dombard and McKinnon 2006. 
+subsection Material model
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+
+    # Reference temperature and viscosity
+    set Reference temperature = 273 
+    set Reference viscosity = 1e11
+
+    # The minimum strain-rate helps limit large viscosities values that arise
+    # as the strain-rate approaches zero.
+    # The reference strain-rate is used on the first non-linear iteration
+    # of the first time step when the velocity has not been determined yet.
+    set Minimum strain rate = 1.e-20
+    set Reference strain rate = 1.e-12
+
+    # Limit the viscosity with minimum and maximum values
+    set Minimum viscosity = 1e10
+    set Maximum viscosity = 1e22
+
+    # Thermal diffusivity is adjusted to match thermal conductivities
+    # assumed in assigning the initial geotherm
+    set Thermal diffusivities =        2.6e-6
+    set Heat capacities       =        1000
+    set Densities             =        930
+    set Thermal expansivities =        0     
+
+    # Harmonic viscosity averaging
+    set Viscosity averaging scheme = harmonic
+
+    # Choose to have the viscosity (pre-yield) follow a dislocation
+    # diffusion or composite flow law.  Here, dislocation is selected
+    # so no need to specify diffusion creep parameters below, which are
+    # only used if "diffusion" or "composite" option is selected.
+    set Viscous flow law = dislocation
+
+    # Dislocation creep parameters for
+    # Note that the viscous pre-factors below are scaled to plane strain from unixial strain experiments.
+    set Prefactors for dislocation creep          =   1.667e-7  
+    set Stress exponents for dislocation creep    =      4.8
+    set Activation energies for dislocation creep =    60e+3
+    set Activation volumes for dislocation creep  =        0
+
+    # Plasticity parameters
+    set Angles of internal friction =   30.
+    set Cohesions                   =  1e7     
+
+  end
+end
+
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 100
+    set Y repetitions = 25
+    set X extent = 100e3
+    set Y extent = 25e3
+  end
+end
+
+subsection Free surface
+  set Free surface boundary indicators        = top
+  set Surface velocity projection = vertical
+end 
+
+# Velocity on boundaries characterized by functions
+# The outward velocity (x-direction) on the left and right walls is 1.25 cm/year
+# The vertical velocity at the base is 0.125 cm/year (balances outflow on sides)
+# Velocity components parallel to the base (x-velocity) and side walls (y-velocity)
+# are unconstrained (i.e. 'free').
+subsection Boundary velocity model 
```

Comment:
Update these comments to reflect that the imposed velocity is now zero. When the values are changed to include shortening, then the comments can be updated again.